docs/api/Pool.md

```diff
@@ -105,3 +105,29 @@ Implements [Client Event: `'disconnect'`](docs/api/Client.md#event-connect)
 ### Event: `'drain'`
 
 Implements [Client Event: `'drain'`](docs/api/Client.md#event-connect)
+
+# Class: RedirectPool
```

Comment:
I think this should be a separate md file.

Reply:
Good point. Moving it.

Reply:
Done!